tests/ui/test_odf_topology.py

```diff
@@ -98,16 +96,19 @@ def test_validate_topology_configuration(
         """
 
         topology_tab = PageNavigator().nav_odf_default_page().nav_topology_tab()
+        random_node_name = random.choice(get_worker_nodes())
+
+        if config.DEPLOYMENT["external_mode"]:
+            if topology_tab.is_entity_present(random_node_name):
+                topology_tab.take_screenshot()
+                pytest.fail(
+                    f"Node {random_node_name} is present in topology, at External mode cluster"
```

Comment:
Why do you use pytest.fail? you can raise exception

Reply:
it looks more clear and pytest - friendly than any Exception.